docs/usage/private-modules.md

```diff
@@ -5,6 +5,120 @@ description: How to support private modules when using Renovate
 
 # Private module support
 
-## npm
+It's a very common requirement to be able to support private module/dependency lookups. This page describes Renovate's approach to authentication.
 
-Please see [Private npm module support](./private-npm-modules)
+First, a quick note on terminology:
+
+- The terms `module`, `package` and `dependency` can mostly be used interchangeably below
+- The terms `credentials`, `secrets` and `authentication` are also used interchangeably
+
+## When does Renovate need credentials?
+
+By default, the only credentials Renovate has are those for the "platform", i.e. GitHub, GitLab, etc. If the token used has sufficient permissions, this will enable Renovate to lookup dependencies located in alternative repositories on the same host or any hosted on any embedded package registry on the same host.
```

Comment:
```suggestion
By default, the only credentials Renovate has are those for the "platform", i.e. GitHub, GitLab, etc.
If the token used has sufficient permissions, this will enable Renovate to lookup dependencies located in alternative repositories on the same host or any hosted on any embedded package registry on the same host.
```